### PR TITLE
Added Mongodb

### PR DESCRIPTION
**Pull request by MalteKiefer**
@kwizart can you take a look?

### CODE REVIEW

File: plugins/mogodb.plugin/uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [ -f "/etc/yum.repos.d/mongodb.repo" ]; then
+  dnf -y remove mongodb-org
```
Comment (kwizart) [inline plugins/mogodb.plugin/uninstall.sh]: You can attempt to remove mongodb-org even if the repo has been deleted in-between.
So you can move this outside of the if condition.

**Comment by kwizart** [comment]
Seems okay.

If we were to generate the changelog automatically, it would be easier if all theses were merged into a single commit with a understandable title, such as "Add mongodb plugin"

Also beware of the remote used to push, you have made a branch into the fedy repository instead of yours. (Nevermind, it can be removed at some point).

**Comment by kwizart** [comment]
Best is to use personal repository to push branches that will be later pushed on fedy/master.

**Comment by MalteKiefer** [comment]
Ok I will close it rebase it and then push it agagin

**Pull request closed without merging**